OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -136,51 +412,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        ገዳማተ፡ ትግሬ፡ ወኢትዮጵያ፡ ብክያ፡ ወላህዋ፡ 
+                                    </q>
+                                    <note>The note is written in Gǝʿǝz with some words and expressions in Amharic. The end of the note is barely legible due to a water stain.</note>
+                                </item>
+                                <item xml:id="a2">
+                                    <desc type="Unclear"></desc>
+                                    <locus target="#78va"/>
+                                    <q xml:lang="am">
+                                        <hi rend="rubric">ሐሰበ<supplied reason="omitted">፡</supplied> ፍከሬሁ<supplied reason="omitted">፡</supplied> 
+                                            ለዳዊት<supplied reason="omitted">፡</supplied> ስምአ<supplied reason="omitted">፡</supplied> ሞተ<supplied reason="omitted">፡</supplied></hi>
+                                        ምሕረት<supplied reason="omitted">፡</supplied> ወንጌላዊ<supplied reason="omitted">፡</supplied> ወርህን<supplied reason="omitted">፡</supplied> 
+                                        በምቶታ<supplied reason="omitted">፡</supplied> ፶ግደፍ፡ ደግሞሞ<supplied reason="omitted">፡</supplied> ሐባርን<supplied reason="omitted">፡</supplied> 
+                                        ሰምን፻፶ግደፍ፡ የወጸውን፡ የዳዊቱን
+                                    </q>
+                                    <note>The note is crudely written in Gǝʿǝz with some words and expressions in Amharic.</note>
+                                </item>
+                                <item xml:id="a3">
+                                    <desc type="GuestText">Miracle of Jesus Christ</desc>
+                                    <locus from="78vb" to="80rb"/>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                        <hi rend="rubric">ተአምረ፡ ዘገ<supplied reason="omitted">ብ</supplied>ረ፡</hi> እግዚእነ፡ ወአምላክነ፡ ኢየሱስ፡ ክርስቶ<supplied reason="omitted">ስ</supplied>
+                                        በኃይለ፡ ሞቱ፡ <sic>ማኀየዊ፡</sic> <add place="above">ይምሐሮ፡ ለ</add>ገብሩ፡ ገብረ፡ ማርያም፡ ወሀሎ፡ ፩ድውይ፡ እንዘ፡ ይግ<add place="above">ዕ</add>ር፡ 
+                                        ወየአውዩ፡ ፲<add place="above">፪</add>ዓመተ፡ ወዘረከቦ፡ በብዝኃ፡ ኃጢአቱ፡ ወተፈትነ፡ በእሳት፡ ከመ፡ ወርቅ፡ ወብሩር፡ መዓልተ፡ ወሌሊተ፡ </seg>
+                                    </q>
+                                    <note>
+                                        The note is hastily written. It narrates a miracle perfomed by Jesus Christ and mentions <persName ref="PRS12983GabraMa" role="patron"/>.
+                                        It might have been written in the same hand as that of <ref target="#a2"/>.
+                                    </note>
+                                </item>
+                                <item xml:id="a4">
+                                    <desc type="GuestText" xml:id="LIT5921SalamLaki"/>
```

Comment:
the correct mark up would be `<desc type="GuestText"><title type="complete/incomplete" ref="LIT5921SalamLaki"/></desc>`. Of course there can be more text in desc than only the title, here for example the content of the note